css/global.css

```diff
@@ -213,8 +226,170 @@ Global Component Styles (buttons, forms, etc.)
 */
 
 /* Buttons */
+.button2 {
+	border-radius: 10px;
+	background-color: var(--button2-color);
+	padding: 10px 40px;
+	color: var(--color-white);
+	border: none;
+	min-width: 187px;
+	min-height: 50px;
+	margin: 20px;
+	font-weight: var(--font-weight-bold);
+	font-size: var(--font21);
+}
 
-/* Forms */
+.button2:hover {
+	background-color: var(--color-white);
+	color: var(--color-onyx);
+}
+
+/* FOOTER STYLES */
+footer {
+	background-color: var(--color-onyx);
+	color: var(--color-white);
+}
+
+footer a img {
+	transition: all 0.1s ease-in;
+}
+footer a img:hover {
+	transform: scale(110%);
+}
+
+.footer-main {
+	padding: 40px 0;
+}
+
+.footer-main {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+.footer-logo-btn {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+/* Land Acknowledgement button */
+.btn-la {
+	background-color: transparent;
+	color: var(--color-white);
+	border: none;
+	margin: 30px 0;
+	font-size: var(--font21);
+}
+
+.btn-la:hover,
+.footer-main ul li a:hover,
+.bottom-links a:hover {
+	border-bottom: 2px solid var(--color-white);
+}
+
+/* FORM Styles */
+form {
+	text-align: center;
+}
+.text-input {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+.text-input input {
+	height: 50px;
+	width: 100%;
+	border-radius: 10px;
+	border: none;
+}
+#newsletter-input {
+	padding-left: 24px;
+}
+
+/* bottom links */
+.bottom-links {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+}
+
+.social-svgs {
+	display: flex;
+	flex-direction: row;
+	column-gap: 20px;
+}
+
+/* MODAL */
+.hidden {
+	display: none;
+}
+/* modal overlay */
+.overlay-la {
+	position: fixed;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	width: 100%;
+	height: 100%;
+	background: rgba(0, 0, 0, 0.5);
+	backdrop-filter: blur(3px);
+	z-index: 1;
+}
+.modal {
+	max-width: 1100px;
+	max-height: 90vh;
+	margin: 0 auto;
+	padding: 26px;
```

Comment:
Modal work looks great! I am wondering if we can use some variables to help out with some  **margin** and **padding** values.